setup.py

```diff
@@ -1,179 +0,0 @@
-import urllib.error
-from distutils import log
-
-from setuptools import find_packages, setup, Command
-from setuptools.command.build_py import build_py as _BuildPyCommand
-from setuptools.command.sdist import sdist as _SDistCommand
-
-package_name = os.environ.get("GALAXY_NG_ALTERNATE_NAME", "galaxy-ng")
-version = "4.11.0dev"
-
-
-class PrepareStaticCommand(Command):
-    DEV_UI_DOWNLOAD_URL = (
-        "https://github.com/ansible/ansible-hub-ui/"
-        "releases/download/dev/automation-hub-ui-dist.tar.gz"
-    )
-
-    ALTERNATE_UI_DOWNLOAD_URL = os.environ.get("ALTERNATE_UI_DOWNLOAD_URL")
-
-    UI_DOWNLOAD_URL = (
-        "https://github.com/ansible/ansible-hub-ui/"
-        f"releases/download/{version}/automation-hub-ui-dist.tar.gz"
-    )
-    TARGET_DIR = "galaxy_ng/app/static/galaxy_ng"
-
-    user_options = [
-        (
-            "force-download-ui",
-            None,
-            "Replace any existing static files with the ones downloaded from github.",
-        ),
-    ]
-
-    def initialize_options(self):
-        self.force_download_ui = False
-
-    def finalize_options(self):
-        pass
-
-    def run(self):
-        if os.path.exists(self.TARGET_DIR):
-            if self.force_download_ui:
-                log.warn(f"Removing {self.TARGET_DIR} and re downloading the UI.")
-                shutil.rmtree(self.TARGET_DIR)
-            else:
-                log.warn(f"Static directory {self.TARGET_DIR} already exists, skipping. ")
-                return
-
-        with tempfile.NamedTemporaryFile() as download_file:
-            log.info(f"Downloading UI distribution to temporary file: {download_file.name}")
-
-            if self.ALTERNATE_UI_DOWNLOAD_URL:
-                log.info(f"Downloading UI from {self.ALTERNATE_UI_DOWNLOAD_URL}")
-                self._download_tarball(self.ALTERNATE_UI_DOWNLOAD_URL, download_file)
-            else:
-                log.info(f"Attempting to download UI for version {version}")
-                try:
-                    self._download_tarball(self.UI_DOWNLOAD_URL, download_file)
-                except urllib.error.HTTPError:
-                    log.warn(f"Failed to retrieve UI for {version}. Downloading latest UI.")
-                    self._download_tarball(self.DEV_UI_DOWNLOAD_URL, download_file)
-
-            log.info(f"Extracting UI static files to {self.TARGET_DIR}")
-            with tarfile.open(fileobj=download_file) as tfp:
-                tfp.extractall(self.TARGET_DIR)
-
-    def _download_tarball(self, url, download_file):
-        urllib.request.urlretrieve(url, filename=download_file.name)
-
-
-class SDistCommand(_SDistCommand):
-    def run(self):
-        self.run_command("prepare_static")
-        return super().run()
-
-
-class BuildPyCommand(_BuildPyCommand):
-    def run(self):
-        self.run_command("prepare_static")
-        return super().run()
-
-
-django_ansible_base_branch = os.getenv('DJANGO_ANSIBLE_BASE_BRANCH', '2025.3.7')
```

Comment:
AFAIK `DJANGO_ANSIBLE_BASE_BRANCH` may be set by some testing pipelines, for example to point to `devel` how would we do it with pyproject?

Reply:
not sure, I need to research more